sdk/extensions/Azure.Extensions.AspNetCore.Configuration.Secrets/README.md

```diff
@@ -1,6 +1,6 @@
-# Azure Storage Blob Key Store for Microsoft.AspNetCore.DataProtection
+# Azure KeyVault Secrets configuration provider for Microsoft.Extensions.Configuration 
 
-The `Azure.Extensions.Configuration.Secrets` package allows storing configuration values using Azure KeyVault Secrets.
+The `Azure.Extensions.AspNetCore.Configuration.Secrets` package allows storing configuration values using Azure KeyVault Secrets.
```

Comment:
"Key Vault" (probably just do a search/replace). Just fresh in memory since once of our vendors did a bunch of fixes recently (day or two ago).